CRM/Report/Form/Event/ParticipantListing.php

```diff
@@ -519,6 +528,14 @@ public function from() {
     $this->joinPhoneFromContact();
     $this->joinEmailFromContact();
 
+    // CRM-20236 Include participant note.
+    if (!empty($this->_params['fields']['participant_note']) || !empty($this->_params['note_value'])) {
```

Comment:
I think the $this->isTableSelected pattern does this & is slightly cleaner / more in keeping with other places. Also not sure the CRM number is important here just because it's not going to give more info if you go & look at it

Reply:
Thanks for the points. I have updated the PR, Please check